pkg/pod/status.go

```diff
@@ -67,6 +67,10 @@ const (
 
 	//timeFormat is RFC3339 with millisecond
 	timeFormat = "2006-01-02T15:04:05.000Z07:00"
+
+	// WarningOverwritingPodAffinity indicates that the affinity specified by
+	// the pod template will be overwritten with affinity assistant
+	WarningOverwritingPodAffinity = "PodAffinityOverwrite"
```

Comment:
Also this, is it possible to declare this in a context so that it can be _private_? I think it is only used in _a single place_  - so I feel it should be possible.

Reply:
Thank you for the review. Fixed.